src/app/test/dot-page-state.service.mock.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { DotRenderedPageState } from '../portlets/dot-edit-page/shared/models/dot-rendered-page-state.model';
+import { mockUser } from './login-service.mock';
+import { DotRenderedPage } from '../portlets/dot-edit-page/shared/models/dot-rendered-page.model';
+import { mockDotLayout, mockDotPage } from './dot-rendered-page.mock';
+
+const mockDotRenderedPage: DotRenderedPage = {
+    page: mockDotPage,
+    html: '',
+    layout: mockDotLayout,
+    canCreateTemplate: true,
+    viewAs: null
+};
+
+@Injectable()
+export class DotPageStateServiceMock {
+    get(url: string): Observable<DotRenderedPageState> {
```

Comment:
But the original get receives an object, right?

Reply:
no really

https://github.com/dotCMS/core-web/blob/master/src/app/portlets/dot-edit-page/content/services/dot-page-state/dot-page-state.service.ts#L45